src/test/java/org/llorllale/cactoos/matchers/InputHasContentTest.java

```diff
@@ -72,4 +73,18 @@ public void describesItself() {
             new TextIs("Input with \"value\"")
         ).affirm();
     }
+
+    @Test
+    public void describesMismatch() {
+        final Description description = new StringDescription();
+        new InputHasContent("world").describeMismatchSafely(
+            new InputOf("input"),
+            description
+        );
+        new Assertion<>(
+            "Can't describe a mismatch",
+            () -> new TextOf(description.toString()),
+            new TextIs("Input with \"input\"")
+        ).affirm();
```

Comment:
@Iprogrammerr I think you should test the Matcher by using it normally and verifying the thrown exception message instead. See #20 for a discussion on this :)

Reply:
@victornoel Sorry, but I can not see how it could be done differently. We are going away from ExpectedException, that is why we have Throws. Using it will require calling matchesSafely directly, which you discouraged in #20.

Reply:
@Iprogrammerr if we can't express it, it means cactoos-matchers is not adequate: you should open an issue and/or add a `@todo` here.
Maybe @llorllale has an opinion?

Reply:
@Iprogrammerr @victornoel we should use `Matches` to test matchers

Reply:
@llorllale great, didn't know this existed :)